python/Ganga/__init__.py

```diff
@@ -299,27 +299,13 @@ def getLCGRootPath():
 lcg_config = makeConfig('LCG', 'LCG/gLite/EGEE configuration parameters')
 #gproxy_config = getConfig('GridProxy_Properties')
 
-# set default values for the configuration parameters
-lcg_config.addOption(
-    'EDG_ENABLE', False, 'DEPRECATED enables/disables the support of the EDG middleware')
-
-lcg_config.addOption('EDG_SETUP', '/afs/cern.ch/sw/ganga/install/config/grid_env_auto.sh',
-                 'DEPRECATED sets the LCG-UI environment setup script for the EDG middleware',
-                 filter=Ganga.Utility.Config.expandvars)
-
-lcg_config.addOption(
-    'GLITE_ENABLE', True, 'Enables/disables the support of the GLITE middleware')
```

Comment:
Just discussed this with @milliams and I'm OK with people seeing some `ConfigError` messages as they're harmless and won't represent a 'bad config' as such (these are just not applicable anymore) but I think we should have a semi-permanent message telling people not to worry about that and pointing them at docs for the new system. If it's just shown on the release notes then people probably won't read it and will only get one chance at seeing it anyway.